leaflet-geoman.d.ts

```diff
@@ -815,6 +833,9 @@ declare module 'leaflet' {
 
             /** angel of rectangle. */
             rectangleAngle?: number;
+
+            /** Cut-Mode: Only the passed layers can be cut. Cutted layers are removed from the Array until no layers are left anymore and cutting is working on all layers again. (Default: []) */
+            layersToCut?: [];
```

Comment:
This will have the same problem as `syncLayersOnDrag` like I mentioned. Should either be like `L.Layer[]` or `(L.Layer|L.SomethingElse)[]` or just `any[]`. Any implies passing things like numbers, or strings is ok, which I'm guessing definitely is not.

Reply:
Ok changed